rollup.config.js

```diff
@@ -1,7 +1,80 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
+import litcss from 'rollup-plugin-lit-css';
+import {terser} from 'rollup-plugin-terser';
+import summary from 'rollup-plugin-summary';
+import replace from '@rollup/plugin-replace';
+
+export function simpleReplace(replacements) {
+  return {
+    name: 'simple-replace',
+    renderChunk(code) {
+      for (const [from, to] of replacements) {
+        code = code.replace(from, to);
+      }
+      return {code};
+    },
+  };
+}
 
 export default [
+  {
+    input: 'src/_codemirror/codemirror-bundle.js',
```

Comment:
Good call. Tried it, but there are some roadbumps that means it doesn't quite work:

- The stock highlighting modes are hard-coded UMD modules that assume a `CodeMirror` global: https://github.com/codemirror/CodeMirror/blob/master/mode/css/css.js

- The `codemirror-grammar-mode` runtime dependency for the `google_modes` has a hard-coded `CodeMirror` global reference: https://github.com/codemirror/grammar-mode/blob/master/src/mode.js#L271

- We can switch `google_modes` Rollup config from UMD to ESM, but it generates an invalid import, because `GrammarMode` is not actually exported by the `codemirror` module, rather it's augmented by `codemirror-grammar-mode`
  ```js
  import { defineMode, GrammarMode } from 'codemirror`;
  ```

These should all be fixable with some more Rollup tricks, but kinda doesn't seem worth it right now.